cluster.go

```diff
@@ -51,7 +51,7 @@ type member struct {
 }
 
 // newCluster returns a new marathon cluster
-func newCluster(client *http.Client, marathonURL string) (*cluster, error) {
+func newCluster(client *http.Client, marathonURL string, DCOSPath string) (*cluster, error) {
```

Comment:
`apiCall()` is not the only place where you need to set the DCOSPath, you also need it in a dozen of places where you make calls to the Marathon API like [this](https://github.com/gambol99/go-marathon/blob/b307c574d90c358369555b8c694ac85cff4e76ac/subscription.go#L40). I modified `newCluster()` because is where all the URL manipulation is done and it would cover all the cases.

I was just trying to keep compatibility with the existing code. I also think the user should set the URL with the **DCOSPath** included. So this is how my new patch would look like:

```Go
-		if r.config.DCOSToken != "" {
-			url = fmt.Sprintf("%s/%s", member+"/marathon", uri)
-		}
```

However, it would break compatibility with the code that @jacob-koren wrote.

@timoreimann please let me know if you want me to submit a new patch to remove the `/marathon` code.